src/java/org/apache/cassandra/db/marshal/CompositeType.java

```diff
@@ -389,4 +470,27 @@ public static <V> V build(ValueAccessor<V> accessor, boolean isStatic, V... valu
         out.flip();
         return accessor.valueOf(out);
```

Comment:
nit: Would it be possible to just replace this w/ `return build(accessor, isStatic, values, (byte) 0);` now?